public/static/locales/en/cognito-errors.json

```diff
@@ -1,13 +1,13 @@
 {
-  "CodeMismatchException": "The security code provided does not match our records.",
+  "CodeMismatchException": "The security code provided does not match our records. Double check your email or request a new code.",
   "InternalServiceException": "We've encountered an error with your request. Please try again later.",
   "InternalServiceExceptionLogin": {
     "title": "There was a problem while signing in.",
     "description": "Try again later or ",
     "link": "/form-builder/support",
     "linkText": "contact Support."
   },
-  "TooManyRequestsException": "Too many requests. Try again later.",
+  "TooManyRequestsException": "Too many requests. Try again later or <a href=\"/en/form-builder/support\" target=\"_blank\">contact Support</a>",
```

Comment:
Note: Need to test this to ensure we can have html markup here.

Reply:
I wasn't sure how to generate a preview of this. Is there a way to test? Or maybe to code the link elsewhere? Alternatively we could keep it more general/broad for now.

Reply:
Tried to look at this yesterday but got sidetracked ... will take a look now.